crates/tx5/Cargo.toml

```diff
@@ -28,6 +28,9 @@ backend-webrtc-rs = [ "tx5-connection/backend-webrtc-rs" ]
 # make a schema available for configuration types
 schema = [ "tx5-connection/schema" ]
 
+# enable some testing utility apis
+test_utils = [ "tx5-connection/test_utils" ]
```

Comment:
There's no naming standard for these but we are more often using kebab case. In Holochain it's a mixture and the test utils are mostly snake case. I think as this case is already using kebab case, I'd prefer kebab case for this feature.

Just checked K2, we managed to keep it consistently kebab case in there too.